projects/storefrontapp-e2e-cypress/cypress/helpers/customer-ticketing/customer-ticketing.ts

```diff
@@ -149,3 +149,77 @@ export function verifyTicketDoesNotExist(ticketDetails: TestTicketDetails) {
 export function visitApparelUKTicketListingPage(){
   visitPage('apparel-uk-spa/en/GBP/my-account/support-tickets', 'apparelTicketListingPage');
 }
+
+export function getTicketDetailsFromTicketListView(){
+  let ticketDetails = [];
+  const row = cy.get('cx-customer-ticketing-list').find('tbody').get('tr').eq(1);
+  row.get('td').eq(0).invoke('text').then((x) => ticketDetails.push(x));
+  row.get('td').eq(1).invoke('text').then((x) => ticketDetails.push(x));
+  row.get('td').eq(5).invoke('text').then((x) => ticketDetails.push(x));
+
+   return ticketDetails;
+}
+
+export function verifyTicketDetails(ticketDetails){
+  //assert title when available
+  cy.get(':nth-child(1) > cx-card > .cx-card > .card-body > .cx-card-container > .cx-card-label-container > :nth-child(1) > .cx-card-label').contains(ticketDetails[1]);
+  cy.get(':nth-child(4) > cx-card > .cx-card > .card-body > .cx-card-container > .cx-card-label-container > :nth-child(1) > .cx-card-label').contains(ticketDetails[2]);
+}
+
+export function createNewTicket(){
+  const testTicketDetails: TestTicketDetails = {
+    subject: 'Entering a subject',
+    message: 'Typing a message',
+    category: TestCategory.complaint,
+  };
+
+  loginRegisteredUser();
+  visitElectronicSupportTicketPage();
+  openCreateTicketModal();
+  fillTicketDetails(testTicketDetails);
+  clickSubmitAndVerifyRequestCompleted();
+  verifyCreatedTicketDetails(testTicketDetails);
+}
+
+export function clickCloseRequestButton(){
+  cy.get(':nth-child(4) > cx-card > .cx-card > .card-body > .cx-card-container > .cx-card-label-container > :nth-child(1) > .cx-card-label').contains("Open");
+  cy.get('.btn.btn-block.btn-action"').click();
+  cy.get('.cx-customer-ticket-form-body').should('exist');
+  cy.get('label > .form-control').last().type("Closing Ticket");
+  cy.get('.cx-customer-ticket-form-footer > .btn-primary').click();
+}
+
+export function verifyStatusOnListingPage(status){
+  const row = cy.get('cx-customer-ticketing-list').find('tbody').get('tr').eq(1);
+  row.get('td').eq(5).contains(status);
+}
```

Comment:
No magic numbers, please.
Make the function name more concrete. Maybe something like verifyStatusOfTicketInPositionAtTicketListingPage(status: string, ticketPosition: number)
Do not restrict yourself from adding long names to JS functions.

Reply:
I think this can work 
```export function verifyClosedTicketIsStillInTicketListing() {
  const row = cy.get('cx-customer-ticketing-list').get('tbody').get('tr').eq(FIRST_ROW_TICKET_LIST);
  row.get('td').eq(STATUS_COLUMN).contains("Closed");
}
```
You can add a parameter if you need as well